airbyte-config/config-persistence/src/main/java/io/airbyte/config/persistence/ConfigRepository.java

```diff
@@ -1544,6 +1544,14 @@ public Geography getGeographyForConnection(final UUID connectionId) throws IOExc
         .fetchOneInto(Geography.class);
   }
 
+  public Geography getGeographyForWorkspace(final UUID workspaceId) throws IOException {
```

Comment:
I don't see this used in this PR, is it called in a different PR or in Cloud? Also probably worth adding a quick unit test for this method

Reply:
Yes - going to be, because at the stage of discover/check we will only have workspace Id available to query which region we should use. Will add a test.